app/assets/javascripts/fae/navigation/_navigation.js

```diff
@@ -32,7 +30,7 @@ Fae.navigation = {
    */
   selectCurrentNavItem: function() {
     var current_base_url = window.location.pathname.replace('#', '');
-    var $ready_current_link = $('.js-nav a[href="' + current_base_url + '"]');
+    var $ready_current_link = $('.js-nav nav a[href="' + current_base_url + '"]');
```

Comment:
@tshedor this selector was highlighting global search results as well. Let me reconfig it.